### PR TITLE
Draft CoC and slack references

### PR DESCRIPTION
**Pull request by joshkoenig**
This PR is a draft work in progress.

### CODE REVIEW

File: source/_docs/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+description: Pantheon is dedicated to a positive and harassment-free community experience for everyone.
+categories: [managing]
+tags: [community]
+contributors:
+  - joshkoenig, rachelwhitton
+date: 9/27/2016
+---
+
+# Contributor Covenant Code of Conduct
+
```
Comment (rachelwhitton) [inline source/_docs/code-of-conduct.md]: Suggest deleting H1 or using H2 instead, [seems a little out of wack as is](https://codeofcon-static-docs.pantheonsite.io/docs/code-of-conduct): 
screen shot 2016-09-27 at 5 05 22 pm

File: source/_docs/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+## Enforcement
+
+Unacceptable behavior from any community member, including maintainers, moderators, Pantheon staff, or others with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, we may take any action deemed appropriate, up to and including a temporary ban or permanent expulsion from the community without warning.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at docs@pantheon.io. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (joshkoenig) [inline source/_docs/code-of-conduct.md]: I think it's ok as-is. Mailto links can be wonky depending on your mail app.

File: source/_docs/code-of-conduct.md (hunk 1 above)
Comment (joshkoenig) [inline source/_docs/code-of-conduct.md]: Probably we should nix the whole thing; it's duplicative with the page title.

**Comment by joshkoenig** [comment]
For now let's leave the CTA alone. This is very much a soft-launch. We'll go bigger with it in a couple weeks. :)

**Comment by hengkit** [comment]
I like the open code of conduct's explicit reference to inclusion based on `socioeconomic status, and technical ability`

**Pull request merged**